### PR TITLE
Feature login

### LINKED ISSUE #5: Login & Signup Page
- [x] Create Login Page
- [x] Create Sign Up Page
- [x] Create Login Form
- [x] Create Signup Form
- [x] Put the forms on the according pages
- [x] Link the 2 pages to each other

### PR DESCRIPTION
**Pull request by inpatr**
closes #5

### CODE REVIEW

**Review by dnmct — CHANGES_REQUESTED** [review]
It's looks odd that the card for login and signup have different sizes.
I think it would look way nicer if they would be the same size.
also consider adding some space between the form and the boundaries of the card.

File: pages/login/index.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                    onChange={(e) => setPassword(e.target.value)}
+                  />
+                </FormControl>
+                <Stack spacing={10} mt={2}>
+                  <Text fontSize="md">
+                    Don&lsquo;t have an account? &nbsp;
+                    <Link onClick={go2oSignup}>SignUp</Link>
+                  </Text>
+                  <Button
+                    type={"submit"}
```
[the remark this answers is not in the document]
Comment (inpatr) [reply pages/login/index.tsx]: removed all unnecessary curlies!

**Comment by inpatr** [comment]
resized signup but with smaller logo:
signup2

okay like that or should we just make both forms way bigger?

**Review by dnmct — CHANGES_REQUESTED** [review]
It's better now.
The logo has different sizes on both pages though.
Also the form can still have more padding, the card still looks cramped

**Comment by inpatr** [comment]
login:
login
signup:
signup

**Review by dnmct — APPROVED** [review]
👍🏻

**Pull request merged**